app/(dashboard)/layout.tsx

```diff
@@ -0,0 +1,18 @@
+'use client'
+
+import Navbar from "../../components/navbar"
+
+export default function DashboardLayout({
+    children, // will be a page or nested layout
+  }: {
+    children: React.ReactNode
+  }) {
+    return (
+        
+      <div className={"flex flex-row justify-start"}>
+        {<Navbar/>}
```

Comment:
Ska inte navbaren ligga i den yttersta foldern så att den syns på alla sidor i dashboard? (Se bild i discord, jag hittar inte wireframen)

Reply:
Det kan den absolut göra, jag fick bara känslan av att vi skulle re-routa till Dashboarden vid inloggning efter clerk

Reply:
Jag trodde clerk liksom låg utanför hela innehållet så att inget av sidan laddas om man inte loggar in? Men om vi ska ha den på alla sidor behöver den nog vara ytterst för nu kommer den ju inte upp på "/"